integration-tests/smoke/ccip/ccip_reader_test.go

```diff
@@ -153,7 +153,7 @@ func emitCommitReports(ctx context.Context, t *testing.T, s *testSetupData, numR
 					},
 				},
 			},
-			MerkleRoots: []ccip_reader_tester.InternalMerkleRoot{
+			BlessedMerkleRoots: []ccip_reader_tester.InternalMerkleRoot{
```

Comment:
Can we have both blessed and unblessed in this test case to make sure we don't have regressions in the chain reader aspect